include/deal.II/multigrid/mg_tools.h

```diff
@@ -198,6 +198,18 @@ namespace MGTools
                       const ComponentMask               &component_mask = ComponentMask());
 
   /**
+  * The same function as above, but return an IndexSet rather than a
+  * std::set<unsigned int> on each level and use a std::set of
+  * boundary_indicators as input
+  */
+  template <int dim, int spacedim>
+  void
+  make_boundary_list (const DoFHandler<dim,spacedim>      &mg_dof,
+                      const std::set<types::boundary_id> &boundary_indicators,
+                      std::vector<IndexSet>                 &boundary_indices,
+                      const ComponentMask               &component_mask = ComponentMask());
```

Comment:
If I am not overlooking something, this would prevent me from calling `make_boundary` multiple times for filling `boundary_indices`. 
Do you suggest to have `boundary_indices` as input argument and return it?